lighthouse-core/config/v2/default.json

```diff
@@ -0,0 +1,308 @@
+      "path": "dobetterweb/anchors-with-no-rel-noopener"
+    },
+    "appcache": {
+      "path": "dobetterweb/appcache"
+    },
+    "console-time-usage": {
+      "path": "dobetterweb/console-time-usage"
+    },
+    "datenow": {
+      "path": "dobetterweb/datenow"
+    },
+    "document-write": {
+      "path": "dobetterweb/document-write"
+    },
+    "geolocation-on-start": {
+      "path": "dobetterweb/geolocation-on-start"
+    },
+    "notification-on-start": {
+      "path": "dobetterweb/notification-on-start"
+    },
+    "domstats": {
+      "path": "dobetterweb/domstats"
+    },
+    "optimized-images": {
+      "path": "dobetterweb/optimized-images"
+    },
+    "tags-blocking-first-paint": {
+      "path": "dobetterweb/tags-blocking-first-paint"
+    },
+    "websql": {
+      "path": "dobetterweb/websql"
+    }
+  },
+
+  "passes": {
+    "defaultPass": {
+      "order": 10,
+      "recordNetwork": true,
+      "recordTrace": true,
+      "pauseBeforeTraceEndMs": 500,
+      "useThrottling": true,
+      "gatherers": [
+        "url",
+        "https",
+        "viewport",
+        "theme-color",
+        "manifest",
+        "accessibility",
+        "image-usage",
+        "content-width"
+      ]
+    },
+    "offlinePass": {
+      "order": 20,
+      "recordNetwork": true,
+      "useThrottling": false,
+      "gatherers": [
+        "service-worker",
+        "offline"
+      ]
+    },
+    "redirectPass": {
+      "order": 30,
+      "useThrottling": false,
+      "gatherers": [
+        "http-redirect",
+        "html-without-javascript"
+      ]
+    },
+    "dbw": {
+      "order": 40,
+      "recordNetwork": true,
+      "useThrottling": false,
+      "gatherers": [
+        "chrome-console-messages",
+        "styles",
+        "css-usage",
+        "all-event-listeners",
+        "anchors-with-no-rel-noopener",
+        "appcache",
+        "console-time-usage",
+        "datenow",
+        "document-write",
+        "geolocation-on-start",
+        "notification-on-start",
+        "domstats",
+        "optimized-images",
+        "tags-blocking-first-paint",
+        "websql"
+      ]
+    }
+  },
+
+  "audits": {
+    "is-on-https": {},
+    "redirects-http": {},
+    "service-worker": {},
+    "works-offline": {},
+    "viewport": {},
+    "manifest-display": {},
+    "without-javascript": {},
+    "first-meaningful-paint": {},
+    "speed-index-metric": {},
+    "estimated-input-latency": {},
+    "time-to-interactive": {},
+    "user-timings": {},
+    "screenshots": {},
+    "critical-request-chains": {},
+    "manifest-exists": {},
+    "manifest-background-color": {},
+    "manifest-theme-color": {},
+    "manifest-icons-min-192": {},
+    "manifest-icons-min-144": {},
+    "manifest-name": {},
+    "manifest-short-name": {},
+    "manifest-short-name-length": {},
+    "manifest-start-url": {},
+    "theme-color-meta": {},
+    "content-width": {},
+    "deprecations": {},
+    "aria-allowed-attr": {
```

Comment:
For audits, as with each of these toplevel objects, the keys are the `id` (currently `audit.meta.name`) of the audit (rather than have it redundantly specified inside the object as well). There's really no need for audits to have category or name in their implementation now either.